examples/07_router/package.json

```
@@ -12,7 +12,7 @@
     "react": "19.0.0-rc-7771d3a7-20240827",
     "react-dom": "19.0.0-rc-7771d3a7-20240827",
     "react-server-dom-webpack": "19.0.0-rc-7771d3a7-20240827",
-    "waku": "0.21.1"
+    "waku": "workspace:*"
```

If it's the same version, it will be the workspace version (unless pnpm changes the behavior).

As this is used as a template the workspace protocol should be avoided.
```suggestion
    "waku": "0.21.1"
```

---

oh, that's cool! I didn't know that.

Just changed it back and you were right, it works just as expected